Source/JavaScriptCore/wasm/WasmIPIntGenerator.cpp

```diff
@@ -756,6 +757,49 @@ PartialResult WARN_UNUSED_RETURN IPIntGenerator::addArguments(const TypeDefiniti
     m_metadata->m_argumINTBytecode.last() = 0x0d;
     return { };
 }
+#elif USE(JSVALUE32_64)
+PartialResult WARN_UNUSED_RETURN IPIntGenerator::addArguments(const TypeDefinition &signature)
+{
+    auto sig = signature.as<FunctionSignature>();
+    auto numArgs = sig->argumentCount();
+    m_metadata->m_numLocals += numArgs;
+    m_metadata->m_numArguments = numArgs;
+    m_metadata->m_argumINTBytecode.resize(numArgs + 1);
+
+    int numGPR = 0;
+    int numFPR = 0;
+
+    // 0x00:        GPR 0/1
```

Comment:
Did your patch getting rid of most of these magic numbers land yet?

Reply:
Yes, these are leftovers TBD. Easier to do once this is upstream tho and I don't have to do it twice for each branch. :)